docs/content/administration/_index.md

```diff
@@ -38,6 +38,19 @@ of enabled apps for guest users in admin settings.
 
 Please note that this enables guest users to create new collectives.
 
+## Searching Collectives
+
+To enable searching of collectives from the unified Nextcloud search, make sure the `ext-pdo` and `ext-pdo_sqlite` PHP extensions are installed and the Nextcloud cronjob is running. The index of collectives page contents should update with every cronjob run. The following commands can be used to install the PHP extensions on Ubuntu:
+
+```
+apt-get install phpXX-mysql # for Nextcloud instances that use MySQL
```

Comment:
I'd rather just list the concrete package (`phpXX-sqlite3`). The paragraph is about searching, so it should only contain the install instructions to get the indexed search working.

Reply:
@mejo-  - I have pushed another commit just listing the packages by name. What I'm not quite sure of though is whether you mean to only list the `phpXX-sqlite3` package? I.e. the `phpXX-mysql` and `phpXX-pgsql` aren't required? If so let me know and I'll remove the mentions of those.

Reply:
Sorry if my comment was missleading. I meant that we shouldn't mention the mysql and pgsql extensions here, as they're unrelated. For the indexed full-text search, only the sqlite extension is relevant. I don't mind keeping it inline as you did now though. Just remove the references to `phpXX-mysql` and `phpXX-pgsql`.

Reply:
Also, please sign your comments (with `git commit --signoff`), so that the DCO test is happy :blush:

Reply:
Got it, thanks! I've updated the PR and signed off the commits :)